test/test_models.py

```diff
@@ -94,7 +94,8 @@ def test_cache_range(self):
         self.assertTrue(c.addr_range[0] <= addr <= c.addr_range[1])
 
     def tearDown(self):
-        os.remove(DEFAULT_DB_PATH)
+        if DEFAULT_DB_PATH != ':memory:':
```

Comment:
`DEFAULT_DB_PATH` is a constant and therefor will always be `":memory:"`.  You'll have to find a different way to determine if the DB was a real file.

Reply:
I don't think it's currently possible to figure this out without significant rewiring.  Go ahead and assume it's always an in-memory database.  We will be able to solve this correctly after we switch to `pytest`.

Reply:
This workaround was meant to help wrap up this issue. The rewiring was going to be worked as part of #24.